cassandra/Dockerfile-5.0.ubi8

```diff
@@ -34,6 +34,55 @@ RUN mkdir -m 775 ${CDC_AGENT_PATH} && \
   ln -s ${CDC_AGENT_EDITION}-${CDC_AGENT_VERSION}-all.jar cdc-agent.jar && \
   chmod -R g+w ${CDC_AGENT_PATH}
 
+FROM --platform=linux/amd64 registry.access.redhat.com/ubi${UBI_MAJOR}/ubi-minimal:${UBI_BASETAG} AS cassandra-builder-amd64
+ARG URL_PREFIX=https://dlcdn.apache.org/cassandra
+ARG CASSANDRA_VERSION
+
+ENV LANG='en_US.UTF-8' LANGUAGE='en_US:en' LC_ALL='en_US.UTF-8'
+ENV CASSANDRA_HOME=/opt/cassandra
+
+# Update base layer
+RUN microdnf install --nodocs shadow-utils \
+    && groupadd -r cassandra --gid=999 \
+    && useradd -m -d "$CASSANDRA_HOME" -r -g cassandra -G root --uid=999 cassandra \
+    && microdnf update && rm -rf /var/cache/yum \
+# Install packages needed during install process
+    && microdnf install tar gzip unzip && microdnf clean all
+
+###
+# Download Cassandra archive
+###
+RUN if test ! -e apache-cassandra-${CASSANDRA_VERSION}-bin.tar.gz; then curl -L -O "https://archive.apache.org/dist/cassandra/${CASSANDRA_VERSION}/apache-cassandra-${CASSANDRA_VERSION}-bin.tar.gz"; fi && \
+    tar --directory ${CASSANDRA_HOME} --strip-components 1 --extract --gzip --file apache-cassandra-${CASSANDRA_VERSION}-bin.tar.gz && \
+    chown -R cassandra:root ${CASSANDRA_HOME} && \
+    chmod -R a+rwX ${CASSANDRA_HOME}
+# MCAC isn't supported on ARM achitectures
+ENV CASSANDRA_PATH=${CASSANDRA_HOME}
+ENV CASSANDRA_CONF=${CASSANDRA_PATH}/conf
+ENV MCAC_PATH=/opt/metrics-collector
```

Comment:
We don't want to ship MCAC with 5.0 builds, it's not going to work.

Reply:
I'm not sure if it works with most 4.1 versions either, but for sure not in 5.0

Reply:
I reverted all the changes to the 5.0 Dockerfile since we don't need them if we will never add MCAC to the image. The removal has already happened in the entrypoint script. I'll leave it in for the 4.1 images, though I think you a re right in that it doesn't work in 4.1.